src/executor/Scheduler.ts

```diff
@@ -18,6 +18,12 @@ interface ISchedulingResult {
 }
 
 export class Scheduler {
+    private static readonly WATCH_PATTERNS = {
+        'ts': 'ts|htm?(l)',
```

Comment:
It actually works like intended: for `ts`, `html` and `htm` files.
Just a question of understanding: what kind of pattern is that? Is the syntax specified somewhere?

Reply:
This is a glob pattern - see: https://github.com/isaacs/node-glob#glob-primer